impl/src/main/java/org/jboss/weld/bootstrap/events/ContainerLifecycleEvents.java

```diff
@@ -200,7 +200,12 @@ private void fireProcessAnnotatedType(ProcessAnnotatedTypeImpl<?> event, Set<Con
             // FastProcessAnnotatedTypeResolver does not consider special scope inheritance rules (see CDI - section 4.1)
             if (checkScopeInheritanceRules(event.getOriginalAnnotatedType(), observer, beanManager)) {
                 try {
-                    observer.notify(event);
+                    //TODO: CDI 1.1 HACK remove once no longer required
```

Comment:
We should add the issue number (WELD-2452) instead of TODO and possibly add some brief description of what the hack actually does...

Reply:
Good point, I actually just took Stuart's commit and added my stuff there so I missed this.